azkaban-common/build.gradle

```diff
@@ -36,7 +36,7 @@ dependencies {
 
   compile('com.google.guava:guava:13.0.1')
   compile('commons-collections:commons-collections:3.2.2')
-  compile('commons-dbcp:commons-dbcp:1.4')
+  compile('org.apache.commons:commons-dbcp2:2.1.1')
```

Comment:
I thought dbcp2 might depend on a specific mysql driver. I tried commenting  
`// compile('mysql:mysql-connector-java:5.1.28')`
Looks like grade doesn't fetch a MySQL driver automatically. Also Checked dbcp's maven specification, [dbcp2](https://mvnrepository.com/artifact/org.apache.commons/commons-dbcp2/2.0) doesn't bring a specified mysql driver. Therefore, this should be a safe change.

Should we also upgrade mysql driver and common-dbutils in later PR?